gateway/api/authentication.py

```diff
@@ -88,6 +90,23 @@ def authenticate(self, request):
                         except User.DoesNotExist:
                             user = User(username=user_id)
                             user.save()
+                    if user_id is None:
+                        logger.warning("No user id.")
+
+                    if not verified:
+                        logger.warning("User is not verified.")
+
+                if verification_data is None:
+                    logger.warning("No verification data returned from request.")
```

Comment:
Some of messages are not user friendly.  What causes no verification data or no authorization data?

Reply:
I added a prefix to the logs that there were problems authenticating to try to make things more clear. Not sure I can be more specific though, I think the goal is just to alert in all the different chances where something might go wrong, though I think we might have to wait until it actually goes wrong to know why it went wrong, if that makes sense